weekly-updates/next.md

```diff
@@ -36,7 +36,10 @@ Solid Events are run by members of the Solid community like you. Solid Event org
 If you have any new [apps](https://github.com/solid/solid-apps), [Pods](https://github.com/solid/pods), or [identity providers](https://github.com/solid/solid-idp-list), or there are updates to existing Solid implementations, make sure to add them to the list so they get included in next week's edition.
 
 ## Solid Development
-(insert updates)
+
+### [Data Browser Project](https://github.com/orgs/solid/projects/4)
+
+A new version of the Data Browser was deployed to https://inrupt.net. It's been available for testing on https://dev.inrupt.net for the while, and is now generally available. This release is another step in the direction of making the Data Browser more approachable to new users: it adds the ability to mark yourself as a Power User or Developer. This will allow us to hide some more advanced functionality for new users, while keeping it available to those to whom it might be relevant. The new option can be found under the new "Preferences" item in your user menu.
```

Comment:
```suggestion
A new version of the Data Browser was deployed to https://inrupt.net. It's been available for testing on https://dev.inrupt.net for a while, and is now generally available. This release is another step toward making the Data Browser more approachable to new users: it adds the ability to mark yourself as a Power User or Developer. This will allow us to hide some more-advanced functionality from new users, while keeping it available to those to whom it might be relevant. The new option can be found under the new "Preferences" item in your user menu.
```